frontend/src/app/constants/constants.d.ts

```diff
@@ -611,6 +611,18 @@ interface PersonFormData extends Person {
   facilityId: string | null;
 }
 
+interface Patient extends PersonFormData {
```

Comment:
I don't think this type definition is correct. The `Patient` in the AoE Form is similar but different then the one stored in the redux state for the patient experience. the two types come from different backend APIs

Reply:
Yes, they come from different API that was a reason to add another extended version on the patientApp.